pyk/src/pyk/ktool/kompile.py

```diff
@@ -320,6 +327,9 @@ def __call__(
         if outer_parsed_json:
             args += ['--outer-parsed-json']
 
+        if ignore_warnings:
+            args += ['-Wno', ', '.join(ignore_warnings)]
```

Comment:
Have you tested this? I thought `-Wno` when used with multiple values had to be specified again, e.g.
`kompile -Wno missing-syntax-module -Wno proof-lint`, not `kompile -Wno missing-syntax-module, proof-lint`

Reply:
Just modified it to add `kompile -Wno missing-syntax-module -Wno proof-lint`

Reply:
However, seems to not make any effect:
![Screenshot from 2024-06-21 21-33-19](https://github.com/runtimeverification/k/assets/38925412/c75729bc-e059-4bbb-94fa-3a4a22783d02)

Reply:
The above seemed to be related to the kevm part of that example. After correcting it, no warnings show this time. I've updated the `-Wno` arguments to be of the form `-Wno A,B,C` since it works equally well and takes less space